examples/simpleapp/src/main/resources/sc_settings.conf

```diff
@@ -55,6 +55,8 @@ scorex {
 
     forger {
         automaticForging = false
+        restrictForgers = false
+        allowedForgersList = []
```

Comment:
do we have somewhere a readme with the syntax to be used inside allowedForgersList ?
if no we should add a comment here

Reply:
we may add a new how-to section to the `examples/simpleapp/README.md` file